msgraph-sdk-raptor-compiler-lib/msgraph-sdk-raptor-compiler-lib.csproj

```
@@ -12,7 +12,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.7" />
     <PackageReference Include="Microsoft.Graph" Version="3.9.0" />
     <PackageReference Include="Microsoft.Graph.Auth" Version="1.0.0-preview.5" />
-    <PackageReference Include="Microsoft.Graph.Beta" Version="0.20.0-preview" />
+    <PackageReference Include="Microsoft.Graph.Beta" Version="0.22.0-preview" />
```

Beta package just got updated.
```suggestion
    <PackageReference Include="Microsoft.Graph.Beta" Version="0.23.0-preview" />
```